library/junos_commit

```diff
@@ -86,6 +86,11 @@ options:
             - Provide a confirm in minutes to the commit of the configuration
         required: false
         default: None
+    check:
+        description:
+            - Do a commit check
```

Comment:
Since the Junos behavior was not obvious to me, can you please add a line or two to this comment explaining why this is useful?  Something like:

Both a `commit` and a `commit check` can be used to confirm a previous `commit confirmed <min>`. However, a `commit check` confirms the previous commit without performing an actual commit and modifying the configuration rollback history.

Reply:
Thanks for the feedback, description updated.